helmfile/charts/prometheus-alerts/values.yaml

```diff
@@ -27,6 +27,7 @@ defaultRules:
     certManagerCertificateAlerts: true
     falcoAlerts: true
     general: true
+    hnc: false
```

Comment:
Do we not want this turned on if we enable hnc?

Reply:
Yes, and that is set up in the values file `prometheus-alerts-sc.yaml.gotmpl` but I forgot to do the same with the Grafana dashboard.

Reply:
Right, I mixed up the values files.